torch/nn/quantized/modules/batchnorm.py

```diff
@@ -61,6 +61,18 @@ def from_float(cls, mod):
         return new_mod
 
 
+class BatchNorm1d(_BatchNorm):
+    r"""This is the quantized version of :class:`~torch.nn.BatchNorm1d`."""
+    _INTRINSIC_BN_RELU = None  # There is no intrinsic for 1d (yet!)
+
+    def __init__(self, *args, **kwargs):
+        super(BatchNorm1d, self).__init__(*args, **kwargs)
+        self.scale = 1.0
+        self.zero_point = 0
```

Comment:
should we make `self.scale` and `self.zero_point` be buffers?  Otherwise the state dict will not correct save/load them.  This problem also exists for the 2d and 3d BNs, but might be good to at least fix it in new files going forward.